tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -339,6 +339,65 @@ public void ShowSchedule_PlayoffScheme_RoundNamesAreCreated()
             CollectionAssert.AreEqual(actual.RoundNames, expected.RoundNames);
         }
 
+        /// <summary>
+        /// Test for ShowSchedule method.
+        /// Valid schedule is passed, no exception occurred.
+        /// </summary>
+        [TestMethod]
+        public void ShowSchedule_PlayoffScheme_TeamsNamesAreAssigned()
```

Comment:
You are trying to cover several cases inside single test. Split it

Reply:
What do you mean by "several"?

Reply:
"TeamsNamesAreAssigned" for non playoff tournaments names are not assigned? make name communicate requirement